code/datums/noobmode.dm

```diff
@@ -0,0 +1,129 @@
+/datum/action/equipHazard
+	name = "Equip skinsuit"
+	desc = "hazard gear and internals. Takes a few seconds."
+	icon_icon = 'icons/obj/clothing/suits.dmi'
+	button_icon_state = "skinsuit"
+	var/obj/item/clothing/suit/space/skinsuit/suitslot
+	var/obj/item/clothing/mask/breath/maskslot
+	var/obj/item/clothing/head/helmet/space/helmetslot
+	var/obj/item/tank/internals/airslot
+	var/obj/item/storage/box/survBox
+	var/paniced = FALSE
+
+/datum/action/equipHazard/Trigger()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		testing(paniced)
+		if(!paniced)
+			if(pannickEquip(H))
+				toggle()
+		else
+			if(stowEquipment(H))
+				toggle()
+		//var/obj/item/held = H.get_active_held_item()
+		/*if(!held)
+			held = H.get_inactive_held_item()
+			if(held && istype(held,/obj/item/storage/box/survival))
+				to_chat(H,"<span class='warning'>You are putting on the outfit inside [held], hold still!</span>")
+				pannickEquip(H)
+			else
+				to_chat(H,"<span class='warning'> you must be holding the box!</span>")
+				return
+		else
+			if(held && istype(held,/obj/item/storage/box/survival))
+				to_chat(H,"<span class='warning'>You are putting on the outfit inside [held], hold still!</span>")
+				pannickEquip(H)
+				*/
+
+
+/datum/action/equipHazard/proc/GatherItems(obj/item/storage/CNT)
+	var/stuff = CNT.contents
+	airslot = locate(/obj/item/tank/internals) in stuff
+	helmetslot = locate(/obj/item/clothing/head/helmet/space) in stuff
+	maskslot = locate(/obj/item/clothing/mask/breath) in stuff
+	suitslot = locate(/obj/item/clothing/suit/space/skinsuit) in stuff
+	if(!airslot || !helmetslot || !maskslot || !suitslot)
+		return FALSE
+	return TRUE
+
+/datum/action/equipHazard/proc/toggle()
+	if(paniced)
+		paniced = 0
+	else
+		paniced = 1
+
+/datum/action/equipHazard/Grant(mob/user, obj/item/containBox)
+	. = ..()
+	survBox = containBox
+
+/datum/action/equipHazard/proc/stowEquipment(mob/living/carbon/human/USR,speed = 10)
```

Comment:
should call the var "user" for consistency

Reply:
I should I dont know propper lingo for bee code, will fix when iam home

Reply:
I wanted to do a replace but realised "usr"(no caps) is used in alot of places. Ill do it anyhow but just fyi

Reply:
yeah usr is a byond var, don't use that 😅